moveit_ros/move_group/src/default_capabilities/tf_publisher_capability.cpp

```diff
@@ -52,6 +53,20 @@ TfPublisher::~TfPublisher()
   thread_.join();
 }
 
+void TfPublisher::publishSubframes(tf2_ros::TransformBroadcaster& broadcaster,
+                                   moveit::core::FixedTransformsMap& subframes, std::string parent_frame)
```

Comment:
CI complains that `parent_frame` should be `const std::string&` instead of `std::string`

Reply:
Also, there is no reason to have this as a member function at all, as it does not use any class variables.
I'd propose to turn this into a local function in anonymous namespace (or at least make the function definition static if you prefer).